sdk/core/core-tracing/lib/tracerProxy.ts

```diff
@@ -23,5 +23,8 @@ export function getTracer() {
   if (!cache.tracer) {
     cache.tracer = new NoOpTracer();
   }
+  if (process.env.AZURE_TRACING_DISABLED) {
```

Comment:
Builds are failing in some places because of the use of `process`, I think `event-hubs` doesn't have its shim set up right.  Might need to wrap this in a `if (isNode) {` block so that it gets elided in the browser builds.

Reply:
it seems like we were trying to check window.__env__ in the browser tho

Reply:
in the logger package we were able to pass `process: false` in `package.json`:

```json
{
"browser": {
    "./dist/index.js": "./browser/logger.js",
    "./dist-esm/src/log.js": "./dist-esm/src/log.browser.js",
    "process": false
  },
}
```

Reply:
maybe that will help with the build breaks?

Reply:
process: false in the package.json browser map, PLUS guarding the process.env check with isNode, is probably the right way to go. I don't think we should support environment variables in the browser.